### PR TITLE
Change asset scale to 9, gwei

### PR DESCRIPTION
**Pull request by kincaidoneil**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -117,7 +162,8 @@ class Plugin extends PluginBtp {
       price
     })
 
-    debug('sending payment.', payment, JSON.stringify(payment))
+    // TODO: how should I debug this?
```
Comment (emschwartz) [inline index.js]: What's this TODO about?

**Review by emschwartz — APPROVED** [review]
This change looks good, but it would be really good to start adding tests for this module. I'm cool with merging this but I'd recommend putting in at least some basic functionality tests before this goes into any kind of real use

**Comment by kincaidoneil** [comment]
I definitely plan to add unit tests, refactor it to be more robust (mainly on the asym-server side: channel watcher, fee logic, convert to TypeScript), and add usable docs/examples in the near future.

**Comment by kincaidoneil** [comment]
Postponing until v2 refactor

**Pull request closed without merging**